docs/rules/no-important.md

```diff
@@ -0,0 +1,21 @@
+# no-important
+
+Disallow `!important` annotations.
+
+## Background
+
+Needing !important indicates there may be a larger underlying issue.
+
+## Rule Details
+
+Examples of incorrect code:
+
+```css
+a {
+	color: red !important;
+}
+
+a .link {
+	font-size: padding: 10px 20px 30px 40px !important;
+}
+```
```

Comment:
Please also add "Prior Art" section as in other rules.

https://stylelint.io/user-guide/rules/declaration-no-important/
https://github.com/CSSLint/csslint/wiki/Disallow-%21important